grg_pssedata/io.py

```diff
@@ -118,7 +118,7 @@ def parse_line(line, line_reqs=None):
 
     if line_reqs is not None:
         if len(line_parts) < line_reqs.min_values:
-            raise PSSEDataParsingError('on psse data line {} in the "{}" section, at least {} values were expected but only {} where found.\nparsed: {}'.format(line_reqs.line_index, line_reqs.section, line_reqs.min_values, len(line_parts), line_parts))
+            raise PSSEDataWarning('on psse data line {} in the "{}" section, at least {} values were expected but only {} where found.\nparsed: {}'.format(line_reqs.line_index, line_reqs.section, line_reqs.min_values, len(line_parts), line_parts))
```

Comment:
Tests are passing now, so I think we can merge this.  One last question.  Why was this changed from `PSSEDataParsingError` to `PSSEDataWarning`?  I think this change will results in a downstream error when the component data structure is constructed.

The changes you made in the `LineRequirements` should be sufficient to support the increased number of default  values.

Reply:
I had thought at first that this solves the problem of incomplete lines. But I forgot to return it to its original state.